package.json

```diff
@@ -51,7 +51,7 @@
       }
     ]
   },
-  "types": "dist/index.d.ts",
+  "types": "package/index.d.ts",
```

Comment:
@danielart This is incorrect. If you run `yarn build` microbundle generates types by default on the path defined here, so `package/index.d.ts` will be replaced by:

```js
declare function createStore(defaultStore: {}, callback: any): any;
declare namespace createStore {
    function ext(extra: any): number;
}
export default createStore;

```

And I think that is not what I want 😅😆

Reply:
didn't know that about microbundle, but the provide a flag called [generateTypes](https://github.com/developit/microbundle#all-cli-options-), so adding this flag negated on build should do the trick. And we update canary version to test it

Reply:
@danielart and @aralroca  just out of curiosity why getting the types from package and not dist, I know those are not generated types, but looking at the main and module tag they both are getting value from dist 
```
  "main": "dist/index.js",
```
We could just do a copy command after the build to copy our custom type, that way everything is bundled in one place.

Reply:
@Chetan-unbounce I agree with your proposal. I also prefer to have everything from the bundle in the same place.